utils/standalone-wizard/remote-kubernetes/remote-pub.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: ${NAMESPACE}
+  name: pub
+  labels:
+    app.kubernetes.io/name: pub
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pub
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: pub
+    spec:
+      serviceAccountName: null
+      securityContext:
+        supplementalGroups: [200000]
+        fsGroup: 200000
+        runAsUser: 100000
+        runAsGroup: 100000
```

Comment:
I wonder if those are needed also when running RT outside Kubernilla context... probably not, but there is a user 100k hardcoded in the dockerfile. I'm not sure if it's bad, ie. perhaps someone could want to run RT in their own hardened K8s env, which enforces other things (like user ids) than our prod env. But I can be completely wrong and perhaps this is neutral (user is created in dockerfile and K8s just uses it). It could be helpful to consult some devops, with more experience in different envs.

Reply:
Even if it's not needed, I think it's still a good practice to use non-root users and groups (just like with Dockerfiles).  So if someone have a need to change it, he probably knows what he's doing and he can modify the files before applying kubectl (Installer asks to review the resources).

Reply:
You're right, if somebody wants to deploy RT to his specific cluster, it's expected that he will need to change few things.